src/lib/pmd/PmdWrapper.ts

```diff
@@ -68,12 +68,15 @@ export default class PmdWrapper extends PmdSupport {
 		// NOTE: If we were going to run this command from the CLI directly, then we'd wrap the classpath in quotes, but this
 		// is intended for child_process.spawn(), which freaks out if you do that.
 		const classpath = await super.buildClasspath();
-		let args = ['-cp', classpath.join(path.delimiter), HEAP_SIZE, MAIN_CLASS, '-rulesets', this.rules, '-dir', this.path,
+		const args = ['-cp', classpath.join(path.delimiter), HEAP_SIZE, MAIN_CLASS, '-dir', this.path,
 			'-format', this.reportFormat];
+		if (this.rules.length > 0) {
+			args.push('-rulesets', this.rules);
+		}
```

Comment:
Is it fine to run without the `-rulesets` parameter if there aren't any rules?

Reply:
If this.rules is blank, it means no category or ruleset was specified...which is fine, right?  All rules are run?  We obviously don't want to pass in --rulesets without a following argument.